lib/measured/conversion.rb

```diff
@@ -1,4 +1,5 @@
 class Measured::Conversion
+  ARBITRARY_CONVERSION_PRECISION = 20
```

Comment:
Yeah, not much to be done here. We have to eventually lose some digits somewhere.